lib/transaction/operation/unfreezing_request.go

```diff
@@ -0,0 +1,48 @@
+package operation
+
+import (
+	"encoding/json"
+
+	"github.com/stellar/go/keypair"
+
+	"boscoin.io/sebak/lib/common"
+)
+
+type UnfreezeRequest struct {
+	Target        string        `json:"target"`
+	Amount        common.Amount `json:"amount"`
+	RequestAmount common.Amount `json:"request-amount"`
+}
+
+func NewUnfreezeRequest(target string, amount common.Amount, request_amount common.Amount) UnfreezeRequest {
```

Comment:
RequestAmount value is not necessary in an unfreeze request. To be unfreeze amount is all or nothing.

Reply:
We discussed it with @soonkuk . While it's not necessary, if we make it a payment operation, it's not possible to differentiate when the request is received whether or not it's a normal payment or an unfreezing, which means we'd have to move one check from `IsWellFormed` to `ValidateOp`. I don't think it's worth it, since unfreezing requests are so rare.

Reply:
@soonkuk That's right. 

Anyway, I don't think `UnfreezeRequest` struct should be a Payable operation. The amount value is always 0 and target is not required, it is just a request.

All frozen accounts are able to send a tx including a payable operation and make unfreeze requests repeatedly. These operations are not blocked yet.